### DISCUSSION TITLE
on-premises registry

### DISCUSSION THREAD

**Discussion by sgalsaleh** [vmware-tanzu/velero#3746] [Community support Q&A]
i’ve got one more question before submitting a new issue: so when using an on-premises registry, is there a way to specify the credentials to use for accessing that registry?

**Comment by EthanLiang1980** [comment]
Hello, You can try it as below, it work for me.

velero install --image 192.168.214.29/tkgm/velero/velero:v1.6.2_vmware.1 \
--provider aws \
--bucket tkg-backup-default \
--secret-file ./credentials-minio \
--namespace velero \
--plugins "192.168.214.29/tkgm/velero/velero-plugin-for-aws:v1.2.1_vmware.1","192.168.214.29/tkgm/velero/velero-plugin-for-vsphere:v1.1.1_vmware.1" \
--backup-location-config region=minio,s3ForcePathStyle="true",s3Url="http://192.168.214.31:9000",publicUrl="http://192.168.214.31:9000" \
--snapshot-location-config region=minio
